Header header
sensor_msgs/PointCloud2 pc1
sensor_msgs/PointCloud2 pc2
sensor_msgs/PointCloud2 pc3
nav_msgs/Odometry pose1
nav_msgs/Odometry pose2
nav_msgs/Odometry pose3
